src/window/win.cc

```diff
@@ -1120,7 +1125,11 @@ namespace SSC {
   }
 
   SSC::String Window::getTitle () {
-    // TODO: implement.
+    LPWSTR title;
+    webview->get_DocumentTitle(&title);
```

Comment:
is it document's title or native window title? should be the latter

Reply:
If that's the case then I'll switch this to use `GetWindowText()`.

Reply:
Fixed.